cirq-google/cirq_google/devices/google_noise_properties_test.py

```diff
@@ -214,7 +228,7 @@ def test_measure_gates():
     op = cirq.measure(*qubits, key='m')
     circuit = cirq.Circuit(cirq.measure(*qubits, key='m'))
     noisy_circuit = circuit.with_noise(model)
-    print(noisy_circuit.moments)
+    # Measurement gates are prepended by amplitude damping, and nothing else.
     assert len(noisy_circuit.moments) == 2
```

Comment:
The prepended noise is the measurement error:
https://github.com/quantumlib/Cirq/blob/f5d8cfb9443907ba47ce7665f418d3231bbe6f1b/cirq-core/cirq/devices/superconducting_qubits_noise_properties.py#L185

We use an amplitude damping channel for this because Cirq doesn't have a "noisy measurement" gate.

Reply:
Got it, thanks!